### PR TITLE
Fix invalid XML id in SAML Request

### PR DESCRIPTION
**Pull request by fenryxo**
Ref #14

### CODE REVIEW

File: eidas_node/proxy_service/views.py [hunk 1]
```diff
@@ -114,7 +113,7 @@ def create_saml_request(self, issuer: str) -> SAMLRequest:
         :return: A SAML request.
         """
         # Replace the request id with the token id to find the request later during pairing with the response.
-        self.light_request.id = self.light_token.id
+        self.light_request.id = 'T' + self.light_token.id  # Prefix with 'T' because XML id cannot start with a digit.
```
Comment (tpazderka) [inline eidas_node/proxy_service/views.py]: The prefix should probably be a constant of some sorts.

File: eidas_node/tests/proxy_service/test_views.py [hunk 2]
```diff
@@ -205,6 +205,20 @@ def test_create_light_response_cannot_find_request(self):
...
+
+    def test_create_light_response_cannot_find_request(self):
+        view = IdentityProviderResponseView()
+        view.request = self.factory.post(self.url)
+        view.storage = IgniteStorage('test.example.net', 1234, 'test-proxy-service-request-cache', '')
+        self.cache_mock.get.return_value = None
+
+        with cast(TextIO, (DATA_DIR / 'saml_response.xml').open('r')) as f:
+            # SAML ID is prefixed with T for cases when the original id starts wit a digit
+            data = f.read().replace('test-saml-request-id', 'Ttest-saml-request-id')
```
Comment (tpazderka) [inline eidas_node/tests/proxy_service/test_views.py]: It is probably better to modify the file itself.

File: eidas_node/proxy_service/views.py (hunk 1 above)
Comment (fenryxo) [reply eidas_node/proxy_service/views.py]: `constants.TOKEN_ID_PREFIX`

File: eidas_node/tests/proxy_service/test_views.py (hunk 2 above)
Comment (fenryxo) [reply eidas_node/tests/proxy_service/test_views.py]: Done.

File: eidas_node/proxy_service/views.py [hunk 3]
```diff
@@ -230,7 +239,7 @@ def create_light_token(self, issuer: str, hash_algorithm: str, secret: str) -> T
         :param secret: A secret shared between communication parties.
         :return: A tuple of the token and its encoded form.
         """
-        token = LightToken(id=str(uuid4()), created=datetime.utcnow(), issuer=issuer)
+        token = LightToken(id=create_xml_uuid('T'), created=datetime.utcnow(), issuer=issuer)
```
Comment (tpazderka) [inline eidas_node/proxy_service/views.py]: You have missed one occurrence :)

Comment (fenryxo) [reply eidas_node/proxy_service/views.py]: Indeed :-)

**Pull request merged**